app/search.py

```diff
@@ -15,12 +15,32 @@ def open_browser(urls):
 
 def search_my_term(term, count):
     return [url for url in search(term, stop=count)]
+def search_rem(term,s,count):
+    return [url for url in search(term,start=s,stop=count)]
 
+def ignore():
+    with open("/home/olagammal/Desktop/search-assist-master/ignore.txt") as f:
```

Comment:
A full URL will fail in others systems, what you can do is create a folder `data` under app and read it from there